coffea/nanoevents/mapping/uproot.py

```diff
@@ -82,6 +85,18 @@ def _extract_base_form(cls, tree, iteritems_options={}):
                         form["content"]["content"]["contents"][field][
                             "form_key"
                         ] = quote(f"{key},!load,!content,!content,{field},!item")
+            elif (
+                form["class"].startswith("ListOffset")
+                and form["content"]["class"] == "RecordArray"
+            ):
+                # TODO: fix me?
```

Comment:
@nsmith- the rest of the uninterpretable items are hitting this new conditional here. I have no idea what magic is going on with `form_key` so I couldn't get this working.